Embryonal subtype update (#152)
AlexsLemonade/OpenPBTA-manuscript: content/03.methods.md

5. If a sample was initially classified as HGAT, had no defining histone mutations, and a BRAF V600E mutation, it was subtyped as `BRAF V600E`.
6. All other high-grade glioma samples that did not meet any of these criteria were subtyped as `HGG, H3 wildtype`.

> Non-MB and non-ATRT embryonal tumors were subset from all embryonal tumor (`Embryonal tumor` in the `broad_histology` column of the metadata pbta-histologies.tsv) using the following criteria:

Reviewer: ```suggestion
Embryonal tumors were included in non-MB and non-ATRT embryonal tumor subtyping if they met any of the following criteria:
```

> 1. An RNA-seq biospecimen sample includes a _TTYH1_ fusion (5' partner).
> 2. An RNA-seq biospecimen sample includes a _MN1_ fusion (5' partner).
> Note that the `MN1--PATZ1` fusion is excluded as it is an entity separate of CNS HGNET-MN1 tumors [@doi:10.1111/nan.12626].

Reviewer: ```suggestion
1. A _TTYH1_ (5' partner) fusion was detected.
2. A _MN1_ (5' partner) fusion was detected, with the exception of `MN1--PATZ1` [@doi:10.1111/nan.12626].
```

> 3. Any sample with "Supratentorial or Spinal Cord PNET" or "Embryonal Tumor with Multilayered Rosettes" in the `pathology_diagnosis` column of the metadata `pbta-histologies.tsv`.

Reviewer: ```suggestion
3. Pathology diagnoses included "Supratentorial or Spinal Cord PNET" or "Embryonal Tumor with Multilayered Rosettes"
```

> 4. Any sample with "Neuroblastoma" in the `pathology_diagnosis` column, where `primary_site` does not contain "Other locations NOS", `pathology_free_text_diagnosis` does not contain "peripheral" or "metastatic".

Reviewer: Does "does not contain 'Other locations NOS'" mean is in the CNS? Because if so, we can just say that I think

```suggestion
4. A pathology diagnosis of "Neuroblastoma", where the tumor was not indicated to be peripheral or metastatic and was located in the CNS.
```

Author: Yes I believe it is so - I changed accordingly!

> 5. Any sample with "Other" in the `pathology_diagnosis` column of the metadata, and with "embryonal tumor with multilayer rosettes, ros (who grade iv)", "embryonal tumor, nos, congenital type", "ependymoblastoma" or "medulloepithelioma" in the `pathology_free_text_diagnosis` column.

Reviewer: I'm not sure the first part (removed below) is a helpful detail to our readers, who we should assume will be unfamiliar with the underlying files.

```suggestion
5. Any sample with "embryonal tumor with multilayer rosettes, ros (who grade iv)", "embryonal tumor, nos, congenital type", "ependymoblastoma" or "medulloepithelioma" in pathology free text.
```

Additional details can be found in the analysis [notebook](https://alexslemonade.github.io/OpenPBTA-analysis/analyses/molecular-subtyping-embryonal/04-table-prep.nb.html).

1. Any RNA-seq biospecimen with _LIN28A_ overexpression, plus a _TYH1_ fusion (5' partner) with a gene adjacent or within the C19MC miRNA cluster and/or copy number amplification of the C19MC region was subtyped as `ETMR, C19MC-altered` (Embryonal tumor with multilayer rosettes, chromosome 19 miRNA cluster altered) [@doi:10.1007/s00401-012-1068-3; 10.1038/ng.2849].
2. Any RNA-seq biospecimen with _LIN28A_ overexpression, a _TTYH1_ fusion (5' partner) with a gene adjacent or within the C19MC miRNA cluster but no evidence of copy number amplification of the C19MC region was subtyped as `ETMR, NOS` (Embryonal tumor with multilayer rosettes, not otherwise specified) [@doi:10.1007/s00401-012-1068-3; @doi:10.1038/ng.2849].
3. Any RNA-seq biospecimen with a fusion having a 5' _MN1_ and 3' _BEND2_ or _CXXC5_ partner were subtyped as `CNS HGNET-MN1` (Central nervous system (CNS) high-grade neuroepithelial tumor with _MN1_ alteration).

> 4. Non-MB and non-ATRT embryonal tumors with internal tandem duplication (as defined in this paper [@doi:10.1186/s12859-016-1031-8]) of _BCOR_ were subtyped as `CNS HGNET-BCOR` (CNS high-grade neuroepithelial tumor with _BCOR_ alteration).

Reviewer: ```suggestion
4. Non-MB and non-ATRT embryonal tumors with internal tandem duplication (as defined in [@doi:10.1186/s12859-016-1031-8]) of _BCOR_ were subtyped as `CNS HGNET-BCOR` (CNS high-grade neuroepithelial tumor with _BCOR_ alteration).
```
